Formula/inform6.rb

```diff
@@ -22,6 +22,11 @@ class Inform6 < Formula
   end
 
   def install
+    if OS.mac?
+      inreplace "Makefile", "-DUNIX", "-DMACOS"
+    else
+      inreplace "Makefile", "-DUNIX", "-DLINUX"
+    end
```

Comment:
We might want to send this patch into `https://gitlab.com/DavidGriffith/inform6unix`

Reply:
Submit a patch into the upstream, https://gitlab.com/DavidGriffith/inform6unix/-/merge_requests/24